mobile/src/main/res/layout/openhabdrawer_menuwithcount.xml

```diff
@@ -19,22 +21,23 @@
         android:scaleType="fitCenter"
         android:layout_marginLeft="20dp"
         android:adjustViewBounds="true"
-        />
+        android:layout_marginStart="20dp" />
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:layout_toRightOf="@+id/itemimage"
         android:layout_marginLeft="20dp"
         android:layout_centerVertical="true"
         android:orientation="horizontal"
-        android:gravity="left|center_vertical"
-        >
+        android:gravity="start|left|center_vertical"
```

Comment:
@maniac103 Does start and left combined also work here?

Reply:
Yes.